src/components/anchor-navigation/anchor-navigation.d.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+export interface AnchorNavigationProps {
+  children?: React.ReactNode;
+  /** The AnchorNavigationItems components to be rendered in the sticky navigation */
+  stickyNavigation?: React.ReactNode;
+  /** Allows to override existing component styles */
+  styleOverride?: {
+   root?: () => object | object;
```

Comment:
So in component we allow `styleOverride` props to be either `func` or `object`. Can we do it here as well maybe?

Reply:
`root?: () => object | object` means that it is either a function returning an object, or just an object